File-Formats/PDF/Convert-HTML-To-PDF/WPF.md

```diff
@@ -10,15 +10,15 @@ keywords: Assemblies
 # Convert HTML to PDF file in WPF
```

Comment:
Use this description - The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or web pages to PDF in Windows Forms application.

Reply:
Added description